.docs/content/1.concepts/4.components.md

```diff
@@ -0,0 +1,75 @@
+
+
+# Project components
+
+This page intends to describe the different software projects and components used for ArmoniK
+internals. To learn how to install or use ArmoniK, please see these repositories:
+  * install:
+  * use:
+
+
+## Common Library
+
+**ArmoniK.Common** provides the components required by all the other compononents of ArmoniK.
```

Comment:
**ArmoniK.Common** provides components required by all the other **components** of ArmoniK.